language/tuning/distilling_step_by_step/Distillation.ipynb

```diff
@@ -0,0 +1,606 @@
+{
```

Comment:
<p>Maybe add an inline link to the PaLM site when you mention it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>With the new intro structure, I have added a link to PaLM2 in the Overview section</p>